### ISSUE TITLE
Task fails with Git Bash on Windows

### ISSUE THREAD

**Issue by Sti2nd** [microsoft/vscode#128682]
The auto-generated build task for typescript/`tsc` does not work on Windows using Git Bash, as mentioned in https://github.com/microsoft/vscode/issues/35593#issuecomment-489015180. There are also mentioned two ways of dealing with, but there is an easier way after #78497 was fixed.

**Solution:** Run tasks in a shell that uses backslash (Windows path separators) as separator by setting the following in your user settings.json:

```json
"terminal.integrated.automationShell.windows": "cmd"
```

This is a duplicate of #36216 and #35593 which are both closed, so created this issue to document.
